changelog/13638.contrib.rst

```diff
@@ -0,0 +1,2 @@
+Fixed deprecated ``gh pr new`` command in :file:`scripts/prepare-release-pr.py`.
+The script now uses ``gh pr create`` which is compatible with GitHub CLI v2.0+.
```

Comment:
From https://github.com/pytest-dev/pytest/pull/13639#discussion_r2262646005, it got away after I renamed this file from  `bugfix` to `contrib`:

```suggestion
Fixed deprecated :command:`gh pr new` command in :file:`scripts/prepare-release-pr.py`.
The script now uses :command:`gh pr create` which is compatible with GitHub CLI v2.0+.
```